assets/app/view/game_page.rb

```diff
@@ -284,7 +284,7 @@ def render_title
     def active_player
       @game_data[:mode] != :hotseat &&
         !cursor &&
-        @game.active_players_id.include?(@user&.dig('id'))
+        @game.current_entity.player.id == @user['id']
```

Comment:
if @user is none this will crash,

so why are we current_entity? it should be active_players

Reply:
In my test there are three active players.  All three got the active banner - that's the bug.  Should it be this?
`@game.current_entity.player.id == @user&.dig('id')`

Reply:
Only one of them is allowed to act, although all 3 get the orange "your turn" banner.  The game operates properly no matter who hits which button ("not your turn" error as appropriate), it's just misleading UI.

Reply:
I made this change and tested some games through it.